### PR TITLE
Fix/form

### PR DESCRIPTION
**Pull request by DmitriyHnatiuk**

### CODE REVIEW

**Review by NiFroPP — APPROVED** [review]

File: src/App/routers/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { User } from 'redux/reducer/userReducer/types';
+import { ways } from 'constants/constRouter';
+
+import Header from 'components/common/Header';
+import Main from 'components/common/Main';
+import Footer from 'components/common/Footer';
+import MainPage from 'pages/MainPage';
+import Error404 from 'pages/Error404';
+import AdminLobby from 'pages/AdminLobby';
+import TeamMembers from 'pages/TeamMembers';
```
Comment (Polina04) [inline src/App/routers/index.tsx]: `import React from 'react';
import { Route, Switch } from 'react-router-dom';

import { useTypedSelector } from 'hooks/useTypedSelector';

import { getMembers } from 'redux/reducer/selectors';
import { User } from 'redux/reducer/userReducer/types';

import { ways } from 'constants/constRouter';

import Header from 'components/common/Header';
import Main from 'components/common/Main';
import Footer from 'components/common/Footer';

import MainPage from 'pages/MainPage';
import Error404 from 'pages/Error404';
import AdminLobby from 'pages/AdminLobby';
import TeamMembers from 'pages/TeamMembers';`

File: src/components/Forms/RegistrationForm/index.tsx [hunk 2]
```diff
@@ -1,48 +1,91 @@
 import React from 'react';
 import { Form, Formik } from 'formik';
```
Comment (Polina04) [inline src/components/Forms/RegistrationForm/index.tsx:3]: redundant empty line

File: src/components/common/PlayerCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import { Users } from 'redux/reducer/userReducer/types';
+
```
Comment (Polina04) [inline src/components/common/PlayerCard/index.tsx:5]: redundant empty line

File: src/pages/AdminLobby/Members/index.tsx [hunk 4]
```diff
@@ -1,15 +1,30 @@
-import React from 'react';
+import PlayerCard from 'components/common/PlayerCard';
+import { useTypedSelector } from 'hooks/useTypedSelector';
+import React, { useMemo } from 'react';
+import { getMembers } from 'redux/reducer/selectors';
+import { User } from 'redux/reducer/userReducer/types';
```
Comment (Polina04) [inline src/pages/AdminLobby/Members/index.tsx:5]: imports order?

File: src/pages/MainPage/index.tsx [hunk 5]
```diff
@@ -1,13 +1,47 @@
...
+
+	const connectToRoom = (event: React.ChangeEvent<HTMLInputElement>): void => {
+		const obj = { roomNumber: event.target.value };
+		dispatch(setUserDataActionCreation(obj));
+	};
+
+	const setAdmin = () => {
+		const obj = { isAdmin: true };
+		dispatch(setUserDataActionCreation(obj));
+		return setForm(true);
```
Comment (Polina04) [inline src/pages/MainPage/index.tsx:35]: please add empty line before this line

File: src/pages/TeamMembers/index.tsx [hunk 6]
```diff
@@ -1,51 +1,109 @@
-import React, { useMemo } from "react";
+import React, { useEffect, useMemo } from 'react';
+import { io } from 'socket.io-client';
```
Comment (Polina04) [inline src/pages/TeamMembers/index.tsx]: redundant empty line

File: src/pages/TeamMembers/index.tsx [hunk 7]
```diff
@@ -1,51 +1,109 @@
...
 				<span>Scram master:</span>
-        <PlayerCard user={admin!} />
+				{admin && <PlayerCard user={admin} />}
+			</div>
+			<div className={styles.exit}>
+				<MyButton value="Exit" onclick={setExit} />
+			</div>
+			<div className={styles.team}>
+				<h3>Members:</h3>
+				<div className={styles.members}>{renderMembers()}</div>
```
Comment (Polina04) [inline src/pages/TeamMembers/index.tsx]: function call in return(

**Review by Polina04 — APPROVED** [review]

**Pull request merged**